usr/share/rear/conf/default.conf

```diff
@@ -1275,17 +1275,25 @@ DUPLY_PROFILE=""
 #
 # BACKUP_DUPLICITY_URL="${DUPLICITY_PROTO}://${DUPLICITY_USER}@${DUPLICITY_HOST}/${DUPLICITY_PATH}"
 #
+# OR
+#
+# Use an NETFS URL
+# BACKUP_URL="cifs://backupserv/md0"
```

Comment:
No, it was Planed to use the already existing `BACKUP_URL` Config Variable if you want to backup to an NETFS-Style URL using duplicity. However if you instead wanted to directly use an Duplicity-Style URL (`BACKUP_DUPLICITY_URL`), that was impossible, because `BACKUP_URL` gets automatically set to `OUTPUT_URL` somewhere. So I later changed the Variable Name (See Commit  bcb4458 and 8d5305d )